ext/bz2/bz2.c

```diff
@@ -367,7 +367,7 @@ static PHP_FUNCTION(bzopen)
 	}
 
 	if (mode_len != 1 || (mode[0] != 'r' && mode[0] != 'w')) {
-		zend_value_error("'%s' is not a valid mode for bzopen(). Only 'w' and 'r' are supported.", mode);
+		zend_argument_value_error(2, "must be a valid mode. Only 'w' and 'r' are supported.");
```

Comment:
Maybe ``"must be 'w' or 'r', \"%s\" given"`` ?

Reply:
We agreed with Nikita that the %s given part shouldn't be added for values (but only for types) since they *might* contain sensitive data (ok, probably not in a gd function, but who knows :) )

Reply:
Fair point

Reply:
There should be no trailing `.` here.